libs/ssl-config/src/test/java/org/opensearch/common/ssl/PemTrustConfigTests.java

```diff
@@ -134,13 +135,40 @@ private void assertCertificateChain(PemTrustConfig trustConfig, String... caName
         assertThat(issuerNames, Matchers.containsInAnyOrder(caNames));
     }
 
+    private void assertFailedToParseOrInvalidFileFormat(PemTrustConfig trustConfig, Path file) {
+        boolean isFailedToParse = false;
+        boolean isInvalidFileFormat = false;
+        try {
+            assertFailedToParse(trustConfig, file);
```

Comment:
Do you know when will `Failed to parse` happen and when will `Invalid format` happen? I think it's better to make the trigger condition clear rather than depending on try...catch.